libnavui-dropin/src/main/res/layout/mapbox_navigation_view_layout.xml

```diff
@@ -52,14 +52,22 @@
                     app:layout_constraintHeight_max="350dp"
                     tools:background="@color/mapbox_main_maneuver_background_color" />
 
+                <FrameLayout
+                    android:id="@+id/scalebarLayout"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    app:layout_constraintTop_toBottomOf="@id/guidanceLayout"
```

Comment:
We don't need to move the scalebar below `maneuverView` when it becomes visible. Let change the constraint to 
```suggestion
                    app:layout_constraintTop_toTopOf="@id/guidanceLayout"
```

Reply:
I take it back. I thought it was landscape.